backend/src/app/rpc/commands/teams.clj

```diff
@@ -80,6 +82,37 @@
   (cond-> row
     (some? features) (assoc :features (db/decode-pgarray features #{}))))
 
+
+
+(defn- check-valid-email-muted
+  "Check if the member's email is part of the global bounce report."
+  [conn member show?]
```

Comment:
This function receives the show? parameter, but looks like it is always `true` so we can simplify it.

Reply:
fixed